src/templates/admin/review/view_review.html

```diff
@@ -82,56 +95,101 @@ <h2>Summary of Review</h2>
-                    {% endif %}
 
-                    {% if review.date_complete %}
-                    <button class="button success" type="submit" name="individual_author_consumption">Update Individual
-                        View Options
-                    </button>
-                    {% endif %}
+                            {% if review.comments_for_editor %}
+                                <div class="large-12 columns">
+                                    <div class="callout" style="padding-bottom:0px;">
+                                        <div class="title-area">
+                                            <h2>Comments for the Editor</h2>
+                                        </div>
+                                        <div class="content">
+                                            <p>{{ review.comments_for_editor|linebreaksbr }}</p>
+                                        </div>
+                                    </div>
+                                </div>
+                            {% endif %}
+                            </div>
+                        {% endif %}
+
                 </div>
-                {% if review.comments_for_editor %}
-                    <div class="title-area">
-                        <h2>Comments for the Editor</h2>
-                    </div>
-                    <div class="content">
-                        <p>{{ review.comments_for_editor|linebreaksbr }}</p>
-                    </div>
-                {% endif %}
-                {% include "elements/review/review_file_table_modal.html" with review=review %}
-            {% endif %}
 
-                {% if review.date_declined %}
+
+                <div class="large-5 columns">
                     <div class="title-area">
-                        <h2>Suggested Reviewers</h2>
+                        <h2>Review Availability Controls</h2>
                     </div>
                     <div class="content">
-                        {{ review.suggested_reviewers|safe }}
+                        <ul class="accordion" data-accordion data-multi-expand="true" data-allow-all-closed="true">
+                            <li class="accordion-item is-active" data-accordion-item>
+                                <!-- Accordion tab title -->
+                                <a href="#" class="accordion-title">Review Visibility</a>
+
+                                <!-- Accordion tab content: it would start in the open state due to using the `is-active` state class. -->
+                                <div class="accordion-content" data-tab-content>
+                                    <p>Currently this review is {{ review.visibility_statement }}.</p>
+                                    <form method="POST">
+                                        {% csrf_token %}
+                                        {{ visibility_form }}
+                                        <button class="button" name="visibility">Save</button>
```

Comment:
I've done this over on the Decision page as it made sense when there is only one toggle per form. I felt for the answers one where there could be a few toggles this might not be ideal but will test out and see how the page responds.